platform/platform-paxweb-jettyconfig/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -75,6 +81,16 @@
 
     <service interface="org.osgi.framework.hooks.service.EventListenerHook" ref="filterInjector"/>
 
-    <reference-list id="filterList" availability="optional" interface="javax.servlet.Filter" />
+    <service id="doSFilter" interface="javax.servlet.Filter" ranking="100">
+        <service-properties>
+            <entry key="filter-name" value="DoSFilter"/>
+            <entry key="urlPatterns" value="/*"/>
+            <entry key="maxRequestsPerSec" value="1000"/>
+            <entry key="delayMs" value="-1" />
+            <entry key="enabled" value="true" />
+            <entry key="ipWhitelist" value="127.0.0.1,0:0:0:0:0:0:0:1" />
+        </service-properties>
+        <bean class="org.eclipse.jetty.servlets.DoSFilter"/>
+    </service>
```

Comment:
Reading the documentation for pax web http://ops4j.github.io/pax/web/6.x/User-Guide.html#WhiteboardExtender-WhatExtenderwilldo: leads me to believe that this should be enough for pax web to pick up the filter and inject it into any Servlets that are published as a service. It doesn't seem to work though so we have to use the filter injector.